Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "B: With an exponential function with values from 0-5, what is the sigma value associated with x = 1? And associated with it, was is its probablity density?"
+    "B: With a standard exponential distrubution function with lambda = 1, what is the sigma value associated with x = 1?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "C: In order to find sigma of x = 1, one must find the what's the value to yield a 50% cdf. Then subtract 1 from the 50% cdf divided by the standard deviation of a exponential distribution. Which is the square root of the (integral from 0 to infinity (x^2)e^-x ) - (integral from 0 to infinity xe^-x)^2. "
```

Comment:
Problem 3C:
- rewording for clarity/explaining to our "audience": How did you come up with the 50% CDF, is it related to the mean being 1? I think the mathematical explanations here could use general editing for audience following.